### DISCUSSION TITLE
github interprets emojis in links

### DISCUSSION THREAD

**Discussion by KiGamji** [community/community#68135] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

hello there! just found out that if there is a link with valid emojis in colons, github will display them, though there is no replacement when opening it, so it's just a visual bug.
an example: https://lore.kernel.org/linux-arm-msm/20230416101134.95686-3-krzysztof.kozlowski@linaro.org/#Z31arch:arm64:boot:dts:qcom:msm8953.dtsi

**Comment by KiGamji** [comment]
just discovered simular issue, but with commit naming: #65755

**Comment by PatCHes992** [comment]
json
